wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js

```diff
@@ -547,13 +576,29 @@
 		 */
 		ajax: function (attrs) {
 			this._initializeDefaults(attrs);
-
 			var res = Wicket.channelManager.schedule(attrs.ch, Wicket.bind(function () {
 				this.doAjax(attrs);
 			}, this));
 			return res !== null ? res: true;
 		},
 
+		/**
+		 * Aborts current AJAX request, if any is running, for default channel.
+		 * WARNING! Mind that this does not implies and server immediately will know about
```

Comment:
This does not imply that the server will immediately know about that? Is it what you're trying to tell?

I think this comment needs rewording.

Reply:
I'd reword it somewhat like this:
```
Abort AJAX request if one in default channel is currently running.

WARNING! Mind that calling this method does not imply that the server will immediately know about the request being aborted.
Server side processing and page are likely to continue.
This method could be useful to cancel huge AJAX requests such as file uploads.
```

Reply:
> This does not imply that the server will immediately know about that? Is it what you're trying to tell?
> 
> I think this comment needs rewording.

What I mean is "there is not reliable way to tell the server stop what I asked you to do". In case of upload stream will be broken and upload halted with a server side exception.

Reply:
@andruhon thanks for your comment.

![image](https://user-images.githubusercontent.com/462655/75333553-fce9c300-588e-11ea-9fb1-c97e53a7b5b3.png)

I have reworded it like above. Hope is is more clear now.